tests/user_restricted/test_user_restricted.py

```diff
@@ -31,16 +33,30 @@ def test_retrieve_deprecated_url(self, headers_with_token):
 
         helpers.check_response_status_code(response, 404)
 
-    # @pytest.mark.smoke_test
+    @pytest.mark.smoke_test
     @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
-    def test_retrieve_patient(self, headers_with_token):
+    @pytest.mark.skipif(ENVIRONMENT != 'int', reason="INT can only use pre-built test app")
+    def test_retrieve_patient_for_int(self,
+                                      apigee_environment,
+                                      nhsd_apim_config,
+                                      _test_app_credentials):
+        headers = helpers.get_headers(apigee_environment,
+                                      nhsd_apim_config,
+                                      _test_app_credentials,
+                                      AUTH_HEALTHCARE_WORKER)
+        self.retrieve_patient_and_assert(retrieve[0], headers)
 
+    @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
+    def test_retrieve_patient_for_non_int(self, headers_with_token):
```

Comment:
Does this need marking?
```python
@pytest.mark.skipif(ENVIRONMENT == 'int')
```

Reply:
Not necessarily. As the tests in INT are defined by the smoke_test marker,  they will never see this test: See https://github.com/NHSDigital/personal-demographics-service-api/blob/dbf7b91767dcfb301d796a9fd4c49809fd753505/azure/templates/pds-tests-int.yml#L49

Reply:
Though actually, is this even a patient by this point? and is it used?

Reply:
Fair enough